app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -63,6 +68,19 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
         .subscribe { uiChange -> uiChange(this) }
   }
 
+  private fun setupApprovalStatusAnimations() {
+    val withInterpolator: (Animation) -> Animation = {
+      it.interpolator = FastOutSlowInInterpolator()
+      it
+    }
+
+    val entryAnimation = AnimationUtils.loadAnimation(context, R.anim.user_approval_status_entry)
```

Comment:
I feel that creating a separate function just for setting the interpolator doesn't really give us much benefit. IMO, either:

```kotlin
entryAnimation.interpolator = FastOutSlowInInterpolator()
viewFlipper.inAnimation = entryAnimation
```

or 

```kotlin
viewFlipper.inAnimation = entryAnimation.apply { interpolator = FastOutSlowInInterpolator() }
```

are better options. What do you think?

Reply:
Sure, done.